### PR TITLE
updates to piecestore stats to report available bandwidth and total used bandwidth

### PR DESCRIPTION
**Pull request by aligeti**
Updated the stats function to report available bandwidth and used bandwidth. The current reporting of bandwidth starts from the beginning of the month to till date

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -121,6 +121,7 @@ require (
 	github.com/garyburd/redigo v1.0.1-0.20170216214944-0d253a66e6e1 // indirect
 	github.com/go-ole/go-ole v1.2.1 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/jinzhu/now v0.0.0-20180511015916-ed742868f2ae
```
Comment (egonelbre) [inline go.mod]: Let's not add trivial dependencies. Every dependency is a liability.

**Comment by aleitner** [comment]
Michal's changes that were just merged also modified the protobuf so you'll need to regenerate

**Comment by aleitner** [comment]
Also update the example code to include the bandwidth info https://github.com/storj/storj/blob/ffddf13f9fea566b64f2e462980b964f0e1b281a/examples/piecestore-client/main.go#L195

File: go.mod (hunk 1 above)
Comment (aligeti) [reply go.mod]: removed the dependency

**Pull request merged**